internal/core/services/events/events_forwarder_service.go

```diff
@@ -180,14 +180,17 @@ func (es *EventsForwarderService) forwardRoomEvent(
 		RoomStatusType: &pingType,
 		Other:          event.Attributes,
 	}
-	err = es.eventsForwarder.ForwardRoomEvent(ctx, roomAttributes, *_forwarder)
+	code, err := es.eventsForwarder.ForwardRoomEvent(ctx, roomAttributes, *_forwarder)
 	if err != nil {
-		reportRoomEventForwardingFailed(scheduler.Game, event.SchedulerID)
-		es.logger.Error(fmt.Sprintf("Failed to forward room events for room %s and scheduler %s", event.RoomID, event.SchedulerID), zap.Error(err))
+		reportRoomEventForwardingFailed(scheduler.Game, event.SchedulerID, string(code))
+		es.logger.Error(fmt.Sprintf("Failed to forward room events for room %s and scheduler %s", event.RoomID, event.SchedulerID),
+			zap.Error(err),
+			zap.Int32("code", code))
+
 		return err
 	}
 
-	reportRoomEventForwardingSuccess(scheduler.Game, event.SchedulerID)
+	reportRoomEventForwardingSuccess(scheduler.Game, event.SchedulerID, string(code))
```

Comment:
What does `string(code)` returns here? From my understanding this is converting an `int32` to string, thus it will report something like `1`, isn't it?